aiida_common_workflows/workflows/relax/cp2k/generator.py

```diff
@@ -243,7 +255,7 @@ def _construct_builder(self, **kwargs) -> engine.ProcessBuilder:
             run_type = 'ENERGY_FORCE'
         else:
             raise ValueError(f'Relax type `{relax_type.value}` is not supported')
-        parameters['GLOBAL'] = {'RUN_TYPE': run_type}
+        parameters['GLOBAL']['RUN_TYPE'] = run_type
```

Comment:
```suggestion
        parameters.setdefault('GLOBAL', {})['RUN_TYPE'] = run_type
```

maybe? On the other hand we probably always require the loglevel set to `MEDIUM` and hence the `GLOBAL` section to be there?

Reply:
Tbh, I am not sure how important the "MEDIUM" print level. Let's keep this change, it is safer like this.